src/main.rs

```diff
@@ -1,3 +1,31 @@
+
 fn main() {
-    println!("Hello, world!");
+    
+    loop{      
+
+        println!("\nQuer adicionar um novo TODO 📝?");
+
+        println!("digite (sim) para confirmar 👍  ou (nao) para negar 👎");
+
+        let resposta= input();
+
+        if resposta  == "sim" {
+            println!("\nQual TODO 📝 deseja criar?");
+
+            let todo : String = input();    
```

Comment:
```suggestion
            let todo = input();    
```

Como a função `input` retorna uma `String`, o compilador consegue _inferir_ o tipo para você, de modo que pode retirar o `: String`, igual você fez no `let resposta = input();` acima.